### PR TITLE
Adding PimpleDi plugin

### PR DESCRIPTION
**Pull request by rochamarcelo**
https://packagist.org/packages/rochamarcelo/cake-pimple-di

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,7 @@ Additional lists you might find useful:
 *Plugins that implement the dependency injection design pattern.*
 
 - [PipingBag plugin](https://github.com/lorenzo/piping-bag) - Dependency injection container plugin that adds the ability to configure object instances and their dependencies before they are used, and to store them into a container class for easy access.
+- [PimpleDi plugin](https://github.com/rochamarcelo/cake-pimple-di) A plugin for dependency injection based on Pimple library
```
Comment (dereuromark) [inline README.md]: Please add a fullstop at the end of the sentence.

Comment (rochamarcelo) [inline README.md]: @dereuromark, okay, file changed with a fullstop.

**Comment by dereuromark** [comment]
Are those two in correct alphabetical order? I think they are not.
After fixing this should be good to go for merging.

**Comment by rochamarcelo** [comment]
Sorry,  now it's in correct alphabetical order.

**Comment by dereuromark** [comment]
Cool. Sry for being dense here ;) I could also fix this post merge.
PS: What do you think of `Allows dependency injection based on Pimple library`. That it is a plugin doesnt need to be repeated, it is already the first words in that line.

**Comment by rochamarcelo** [comment]
Sounds good.

**Pull request merged**